core/src/env/srml/srml_only/impls.rs

```diff
@@ -86,10 +86,12 @@ where
     T: EnvTypes,
 {
     type AccountId = <T as EnvTypes>::AccountId;
+    type AccountIndex = <T as EnvTypes>::AccountIndex;
```

Comment:
Btw.: Why is `AccountId` no longer enough? As far as I know SRML contracts doesn't even really support `AccountIndex` since it would require a 2-phase runtime call in order to be used compared to a `AccountId`.

Reply:
I added this because the `srml-balances` module (and other srml modules) expect the `Address` enum as an argument (either Id or Index).

You are correct, and I have removed `AccountIndex` from `EnvTypes`. However as part of https://github.com/paritytech/ink-types-node-runtime/pull/3, you are still able to construct an `Address::Index` to send as part of a `Call` into the runtime.